package-lock.json

```diff
@@ -5,14 +5,15 @@
   "packages": {
     "": {
       "dependencies": {
-        "@nationalarchives/frontend": "0.2.14"
+        "@nationalarchives/frontend": "0.2.14",
+        "psycopg2-binary": "^0.0.1"
```

Comment:
I don't think this should be in here. `psycopg2-binary` is a Python Postgres adapter.

Reply:
I've removed that entry.